src/libraries/Native/native-binplace.proj

```diff
@@ -1,9 +1,9 @@
 <Project DefaultTargets="Build">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp;
+      netcoreapp5.0;
```

Comment:
Should this one use `NetCoreAppCurrent`? Or this doesn't import the root `Directory.Build.props`?

Reply:
If so, I believe we should use it to mitigate errors when we move to the next version of netcoreapp.